clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/SearchDAO.scala

```diff
@@ -45,18 +42,10 @@ trait SearchDAO {
   /**
     * Query a metadata index.
     *
-    * @param input       An object describing the query to run.
+    * @param queryDefinition       The query to run.
     * @param index       The index to run the query against.
-    * @param queryMapper Utility for mapping the query input to an
-    *                    elastic4s query object, and for mapping the
-    *                    query results back to a query output type.
-    * @tparam I          The type of the query input.
-    * @tparam O          The type of the query output.
-    * @tparam D          The type of the document to query.
+    * @tparam D The type of the document to query.
```

Comment:
Also here.

Reply:
Personally I think lining these up is a waste of time, but I may as well preserve the current file's convention.